http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1728,6 +1728,45 @@ paths:
                 items:
                   $ref: "#/components/schemas/ListAppResponse"
 
+  "/api/apps/v2/{workspaceId}/deleteAll":
+    post:
```

Comment:
Yeah, this is what Rob recommended. DELETE is for a single entity, and PATCH isn't quite right either since no resources are modified. POST is a generic catch-all for 'modify resources with side effects'